src/+otp/+kuramotosivashinsky/javp.m

```diff
@@ -0,0 +1,5 @@
+function jv = javp(~, u, k, k2, k4, v)
```

Comment:
The problem parameters are always the last argument.  It should be `(~, u, v, k, k2, k4)`